### ISSUE TITLE
Nuget package ?

### ISSUE THREAD

**Issue by julienadam** [cyotek/Cyotek.Collections.Generic.CircularBuffer#1]
I'd love to reference this through a nuget package, would it be possible to create one ? You already have a build running on AppVeyor so it should be quite easy to push packages from there.

Thanks for this implentation, the original on codeplex has indeed quite a few bugs and I was about to fix it when I found this one :)

**Comment by cyotek** [comment]
You're welcome, hope you found it useful. And I hope I fixed the bugs that you've found - if you do find anything remiss feel free to PR or raise an issue :)

As for the package - I'll take a look at it this weekend and create one, I must admit I'm trying to use NugGet for everything myself these days. With my other OS projects I have a batch file to generate the packages, this was the first project I tried using AppVeyor with - but if they can generate the package for me, that's probably preferable so I'll give it a go.

Regards;
Richard Moss
